FAILED tests/test_cli_help_py312.txt::test_cli_help_py312.txt

Hello,

during the packaging of 3.6.0 on Gentoo Linux I encountered the following test failure:

```
python3.12 -m pytest -vv -ra -l -Wdefault -Werror::pytest.PytestUnhandledCoroutineWarning --color=yes -o console_output_style=count -o tmp_path_retention_count=0 -o tmp_path_retention_policy=failed -p no:cov -p no:flake8 -p no:flakes -p no:pylint -p no:markdown -p no:sugar -p no:xvfb -p no:pytest-describe -p no:plus -p no:tavern -p no:salt-factories -k not test_ui.txt and not test_data.py
============================= test session starts ==============================
platform linux -- Python 3.12.8, pytest-8.3.5, pluggy-1.5.0 -- /var/tmp/portage/dev-db/pg_activity-3.6.0/work/pg_activity-3.6.0-python3_12/install/usr/bin/python3.12
cachedir: .pytest_cache
psycopg: 3.2.7
rootdir: /var/tmp/portage/dev-db/pg_activity-3.6.0/work/pg_activity-3.6.0
configfile: pytest.ini
collecting ... collected 80 items / 17 deselected / 63 selected

pgactivity/activities.py::pgactivity.activities.sorted PASSED           [ 1/63]
pgactivity/activities.py::pgactivity.activities.update_max_iops PASSED  [ 2/63]
pgactivity/config.py::pgactivity.config.Configuration.parse PASSED      [ 3/63]
pgactivity/config.py::pgactivity.config.Flag PASSED                     [ 4/63]
pgactivity/data.py::pgactivity.data.Data.get_duration_column PASSED     [ 5/63]
pgactivity/data.py::pgactivity.data.pg_get_short_version PASSED         [ 6/63]
pgactivity/handlers.py::pgactivity.handlers.duration_mode PASSED        [ 7/63]
pgactivity/handlers.py::pgactivity.handlers.query_mode PASSED           [ 8/63]
pgactivity/handlers.py::pgactivity.handlers.refresh_time PASSED         [ 9/63]
pgactivity/handlers.py::pgactivity.handlers.sort_key_for PASSED         [10/63]
pgactivity/handlers.py::pgactivity.handlers.wrap_query PASSED           [11/63]
pgactivity/keys.py::pgactivity.keys._sequence_by_int PASSED             [12/63]
pgactivity/types.py::pgactivity.types.Column PASSED                     [13/63]
pgactivity/types.py::pgactivity.types.SelectableProcesses PASSED        [14/63]
pgactivity/types.py::pgactivity.types.SystemInfo.default PASSED         [15/63]
pgactivity/types.py::pgactivity.types.UI.column PASSED                  [16/63]
pgactivity/types.py::pgactivity.types.UI.columns PASSED                 [17/63]
pgactivity/types.py::pgactivity.types.UI.end_interactive PASSED         [18/63]
pgactivity/types.py::pgactivity.types.UI.evolve PASSED                  [19/63]
pgactivity/types.py::pgactivity.types.UI.start_interactive PASSED       [20/63]
pgactivity/types.py::pgactivity.types.UI.tick_interactive PASSED        [21/63]
pgactivity/types.py::pgactivity.types.UI.toggle_pause PASSED            [22/63]
pgactivity/types.py::pgactivity.types.UIHeader.make PASSED              [23/63]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_instance PASSED   [24/63]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_system PASSED     [25/63]
pgactivity/types.py::pgactivity.types.UIHeader.toggle_workers PASSED    [26/63]
pgactivity/types.py::pgactivity.types.enum_next PASSED                  [27/63]
pgactivity/utils.py::pgactivity.utils.MessagePile PASSED                [28/63]
pgactivity/utils.py::pgactivity.utils.clean_str PASSED                  [29/63]
pgactivity/utils.py::pgactivity.utils.csv_write PASSED                  [30/63]
pgactivity/utils.py::pgactivity.utils.ellipsis PASSED                   [31/63]
pgactivity/utils.py::pgactivity.utils.format_duration PASSED            [32/63]
pgactivity/utils.py::pgactivity.utils.get_duration PASSED               [33/63]
pgactivity/utils.py::pgactivity.utils.short_state PASSED                [34/63]
pgactivity/utils.py::pgactivity.utils.wait_status PASSED                [35/63]
pgactivity/utils.py::pgactivity.utils.yn PASSED                         [36/63]
pgactivity/views.py::pgactivity.views.format_query PASSED               [37/63]
pgactivity/views.py::pgactivity.views.get_indent PASSED                 [38/63]
pgactivity/views.py::pgactivity.views.limit PASSED                      [39/63]
pgactivity/views.py::pgactivity.views.query_mode PASSED                 [40/63]
pgactivity/views.py::pgactivity.views.shorten PASSED                    [41/63]
tests/test_activities.py::test_ps_complete PASSED                       [42/63]
tests/test_activities.py::test_ps_complete_empty_procs PASSED           [43/63]
tests/test_activities.py::test_mem_swap_load PASSED                     [44/63]
tests/test_cli.py::test_parser PASSED                                   [45/63]
tests/test_cli.py::test_parser_flag_on PASSED                           [46/63]
tests/test_cli_help.txt::test_cli_help.txt SKIPPED (only applies for
Python version >= 3.13)                                                 [47/63]
tests/test_cli_help_py312.txt::test_cli_help_py312.txt FAILED           [48/63]
tests/test_config.py::test_flag PASSED                                  [49/63]
tests/test_config.py::test_flag_load PASSED                             [50/63]
tests/test_config.py::test_error PASSED                                 [51/63]
tests/test_config.py::test_lookup PASSED                                [52/63]
tests/test_config.py::test_lookup_builtin_profiles[minimal-expected0] PASSED [53/63]
tests/test_config.py::test_lookup_builtin_profiles[narrow-expected1] PASSED [54/63]
tests/test_config.py::test_lookup_builtin_profiles[wide-expected2] PASSED [55/63]
tests/test_scroll.txt::test_scroll.txt PASSED                           [56/63]
tests/test_types.py::test_filters_from_options PASSED                   [57/63]
tests/test_views.py::test_columns_header[pid+database; sort by cpu; activities] PASSED [58/63]
tests/test_views.py::test_columns_header[cpu; sort by cpu; activities] PASSED [59/63]
tests/test_views.py::test_columns_header[mem; sort by cpu; activities] PASSED [60/63]
tests/test_views.py::test_columns_header[many flags; sort by duration; blocking] PASSED [61/63]
tests/test_views.txt::test_views.txt PASSED                             [62/63]
tests/test_widgets.txt::test_widgets.txt PASSED                         [63/63]

=================================== FAILURES ===================================
______________________ [doctest] test_cli_help_py312.txt _______________________
001 >>> import sys
002 >>> import pytest
003 >>> if sys.version_info >= (3, 13):
004 ...     pytest.skip("only applies for Python version < 3.13")
005 ...
006 
007 >>> from pgactivity import cli
008 >>> parser = cli.get_parser()
009 >>> parser.print_help()
Differences (unified diff with -expected +actual):
    @@ -1,3 +1,3 @@
    -usage: pytest [options] [connection string]
    +usage: __main__.py [options] [connection string]
     <BLANKLINE>
     htop like application for PostgreSQL server activity monitoring.

/var/tmp/portage/dev-db/pg_activity-3.6.0/work/pg_activity-3.6.0/tests/test_cli_help_py312.txt:9: DocTestFailure
=========================== short test summary info ============================
SKIPPED [1] <doctest test_cli_help.txt[2]>:2: only applies for Python version >= 3.13
FAILED tests/test_cli_help_py312.txt::test_cli_help_py312.txt
============ 1 failed, 61 passed, 1 skipped, 17 deselected in 0.21s ============
```

If I can help further, please let me know.

--- a/pgactivity/cli.py
+++ b/pgactivity/cli.py
@@ -56,8 +56,9 @@ def flag(p: Any, spec: str, *, dest: str, feature: str) -> None:
     )
 
 
-def get_parser() -> argparse.ArgumentParser:
+def get_parser(prog: str | None = None) -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(
+        prog=prog,
         usage="%(prog)s [options] [connection string]",
         description=(
             "htop like application for PostgreSQL server activity monitoring."

--- a/tests/test_cli_help.txt
+++ b/tests/test_cli_help.txt
@@ -5,9 +5,9 @@
 ...
 
 >>> from pgactivity import cli
->>> parser = cli.get_parser()
+>>> parser = cli.get_parser(prog="pg_activity")
 >>> parser.print_help()
-usage: pytest [options] [connection string]
+usage: pg_activity [options] [connection string]
 <BLANKLINE>
 htop like application for PostgreSQL server activity monitoring.
 <BLANKLINE>

--- a/tests/test_cli_help_py312.txt
+++ b/tests/test_cli_help_py312.txt
@@ -5,9 +5,9 @@
 ...
 
 >>> from pgactivity import cli
->>> parser = cli.get_parser()
+>>> parser = cli.get_parser(prog="pg_activity")
 >>> parser.print_help()
-usage: pytest [options] [connection string]
+usage: pg_activity [options] [connection string]
 <BLANKLINE>
 htop like application for PostgreSQL server activity monitoring.
 <BLANKLINE>

